### PR TITLE
add version.rb and remove Gem.loaded_specs

### PR DESCRIPTION
**Pull request by TheRoyalTnetennba**

### CODE REVIEW

**Comment by TheRoyalTnetennba** [comment]
need to update Gemfiles

File: google-cloud-error_reporting/lib/google/cloud/error_reporting/v1beta1/error_group_service_client.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 require "google/devtools/clouderrorreporting/v1beta1/error_group_service_pb"
 require "google/cloud/error_reporting/v1beta1/credentials"
-require "google/cloud/errorreporting/version"
+require "google/cloud/error_reporting"
```
Comment (blowmage) [inline google-cloud-error_reporting/lib/google/cloud/error_reporting/v1beta1/error_group_service_client.rb]: I think this should be `require "google/cloud/error_reporting/version"`

File: google-cloud-datastore/lib/google/cloud/datastore.rb [hunk 2]
```diff
@@ -18,7 +18,7 @@
 require "google/cloud/datastore/dataset"
 require "google/cloud/datastore/transaction"
 require "google/cloud/datastore/credentials"
-require "google/cloud/config"
+require "google/cloud/config/version"
```
Comment (blowmage) [inline google-cloud-datastore/lib/google/cloud/datastore.rb]: This is wrong.

**Comment by blowmage** [comment]
Woo! \o/ This looks great! Can't wait to see how CI does.

**Comment by TheRoyalTnetennba** [comment]
@blowmage do you feel this is good to go?

**Comment by blowmage** [comment]
Yes, looking forward to having this merged.

**Pull request merged**